app/models/manageiq/providers/storage_manager/swift_manager/refresh_parser/cross_linkers.rb

```diff
@@ -5,6 +5,10 @@ def self.cross_link(ems, data)
       _log.warn "Manager does not have a parent."
       return
     end
+    unless data
+      _log.warn "Manager does not have any object storage."
```

Comment:
@Fryguy See http://www.rubydoc.info/github/bbatsov/rubocop/Rubocop/Cop/Lint/RequireParentheses
They only seem concerned about having parenthesis if there is `&&` or `||`. :cry: